app/policies/article_policy.rb

```diff
@@ -2,28 +2,71 @@
 
 class ArticlePolicy < ApplicationPolicy
   def index?
-    user.root? or user.editor?
+    user.root? or user.editor? or user.neighbourhood_admin? or user.partner_admin?
```

Comment:
this line doesnt do anything and this function has no implicit return

Reply:
Grhhh. Copied it over from my stash wrong :woman_facepalming: